cfme/infrastructure/virtual_machines.py

```diff
@@ -1180,10 +1180,9 @@ def resetter(self, *args, **kwargs):
 @navigator.register(Vm, 'Details')
 class VmAllWithTemplatesDetails(CFMENavigateStep):
     VIEW = InfraVmDetailsView
-    prerequisite = NavigateToSibling('All')
```

Comment:
Q: What is it for ?

Reply:
Changed to list only provider vm as came across failures "ManyEntitiesFound" as also orphaned and archived vms from removed providers were on the list.